src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -62,17 +58,16 @@ class CardsResource extends Resource {
     @GET
     @Path ('{id}')
     @Produces(MediaType.APPLICATION_JSON)
-    Response getCardById(@PathParam('id') IntParam id) {
+    Response getCardById(@Auth AuthenticatedUser authenticatedUser, @PathParam('id') IntParam id) {
```

Comment:
feel free to leave it the way it is, but you can also do a `@PermitAll` annotation on the class to require authentication for all the `Response` methods. Example: https://github.com/osu-mist/hr-api/blob/master/src/main/groovy/edu/oregonstate/mist/hr/resources/HRResource.groovy#L26

It might be nicer when you add more methods.

Reply:
- ResourceObject method now uses already existing Card object
- Authentication now used for all methods instead of just GET /cards/{id}